CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## dev
 
+## 2/14/23
+* Add RnaSeqSpliceOutlier model (REQUIRES DB MIGRATION)
```

Comment:
changelog entries go in the `dev` section, I will handle updating the release date when we actually do a release